src/parser/flags.ts

```diff
@@ -40,6 +40,25 @@ export const integer = build({
   },
 })
 
+export const url = build({
+  parse: async input => {
+    try {
+      return new URL(input)
+    } catch {
+      throw new Error(`Expected a valid url but received ${input}`)
+    }
+  },
+})
+
+export const filepath = build({
```

Comment:
What if we just called it `path`? That way it doesn't matter if it is a file or directory?

I'm also thinking what else we could do here that would be interesting to make it a little more helpful than just string. For example, wanting a path that defaults to the cwd is super useful, but the help gets all messed up. Maybe that is something this flag could fix? i.e. if `default: <cwd>` then automatically initialize the value to `process.cwd()` or something like that?

On that note, maybe these should have description headers comments?

Reply:
@amphro Is there a specific reason that you don't want `process.cwd()` to show up in help?

```~/code/plugin-login [main] : bin/dev login org jwt --help
Log in to your Salesforce orgs using a JSON web token

USAGE
  $ sf login org jwt

OPTIONS
  -a, --alias=<value>         Set an alias for the account or environment
  -d, --set-default           Set the org as the default org after login
  -f, --jwt-key-file=<value>  [default: /Users/mdonnalley/code/plugin-login] Path to a file containing the private key
  -i, --clientid=<value>      OAuth client ID (sometimes called the consumer key)
  -l, --instance-url=<value>  [default: https://login.salesforce.com] The login url
  -u, --username=<value>      Authentication username

EXAMPLE
  sf login org jwt --jwt-key-file myorg.key --username me@salesforce.com --clientid XXXX
```

I personally think it's pretty cool that it shows me the exact path it'll use as the default

Reply:
Because help is cached in `oclif.manifest`. So the end result is it will show the path of whatever machine generated the `oclif.manifest`, which is usually the build machine.

Reply:
Oh I see. Okay, I'll fix that up